data/sql/updates/pending_db_world/Fix_Fel_Cannon_MKI.sql

```diff
@@ -0,0 +1,2 @@
+
+UPDATE `creature_template` SET `unit_flags` = `unit_flags`|4 WHERE (`entry` = 22461);
```

Comment:
This does *not* match sniffs. 3.4.0 45854:

> Flags: 0
> Flags2: 2048

Reply:
Ok. I Will try to find another way.